app/dashboard/views.py

```diff
@@ -758,25 +758,38 @@ def users_fetch(request):
     skills = request.GET.get('skills', '')
     limit = int(request.GET.get('limit', 10))
     page = int(request.GET.get('page', 1))
-    order_by = request.GET.get('order_by', '-actions_count')
+    order_by = request.GET.get('order_by', 'leaderboard_ranks__rank')
     bounties_completed = request.GET.get('bounties_completed', '').strip().split(',')
     leaderboard_rank = request.GET.get('leaderboard_rank', '').strip().split(',')
     rating = int(request.GET.get('rating', '0'))
     organisation = request.GET.get('organisation', '')
 
     user_id = request.GET.get('user', None)
     if user_id:
-        profile = Profile.objects.get(id=int(user_id))
+        profile = User.objects.get(id=int(user_id))
     else:
-        profile = request.user.profile if hasattr(request, 'user') and request.user.is_authenticated and hasattr(request.user, 'profile') else None
+        profile = request.user if hasattr(request, 'user') and request.user.is_authenticated else None
 
     context = {}
     if not settings.DEBUG:
         network = 'mainnet'
     else:
         network = 'rinkeby'
 
-    user_list = Profile.objects.prefetch_related('fulfilled', 'leaderboard_ranks', 'feedbacks_got').order_by(order_by)
+
+    user_list = Profile.objects.prefetch_related(
```

Comment:
similarly here, can we can this `profile_list`?

Reply:
Hmm. I mean it's called User Directory. So I thought user_list is good. 🤔